backend/support/management/commands/run_2022.py

```diff
@@ -26,6 +26,7 @@ def handle(self, *args, **kwargs):
             sac = self.make_sac(gen)
             sac.submission_status = sac.STATUS.SUBMITTED
             sac.save()
+            sac.disseminate()
```

Comment:
run_2022 is just a proram to check our logic. We are not actually determining cog/over for 2022 as these were done by census.  I don't think we need calll disseminate here. In fact, we need to make sure we don't run this code in any env other than local, or perhaps dev.